include/mbedtls/mbedtls_config.h

```diff
@@ -0,0 +1,67 @@
+/**
+ * \file mbedtls_config.h
+ *
+ * \brief Configuration options (includable header)
+ *
+ *  Include this file if you need to depend on the
+ *  configuration options defined in mbedtls_config_defines.h
+ */
+ /*
+  *  Copyright The Mbed TLS Contributors
+  *  SPDX-License-Identifier: Apache-2.0
+  *
+  *  Licensed under the Apache License, Version 2.0 (the "License"); you may
+  *  not use this file except in compliance with the License.
+  *  You may obtain a copy of the License at
+  *
+  *  http://www.apache.org/licenses/LICENSE-2.0
+  *
+  *  Unless required by applicable law or agreed to in writing, software
+  *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  *  See the License for the specific language governing permissions and
+  *  limitations under the License.
+  */
+
+#ifndef MBEDTLS_MBEDTLS_CONFIG_H
```

Comment:
Yes, I've duplicated `MBEDTLS_`, since our convention for header include guards has been, from what I can gather `NAME_OF_FOLDER_NAME_OF_HEADER`. So `mbedtls/mbedtls_c.onfig.h` translates to an include guard of `MBEDTLS_MBEDTLS_CONFIG_H`